nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```
@@ -632,6 +632,7 @@
                         <exclude>src/test/resources/TestConvertRecord/input/person_dropfield.json</exclude>
```

Not sure why this is not raised by the RAT check but this should be updated.

```suggestion
                        <exclude>src/test/resources/TestConvertRecord/input/personJob_dropfield.json</exclude>
```

---

@pvillard31 Good catch! I just made the change.